configuration/pih/liquibase/liquibase.xml

```diff
@@ -120,6 +123,12 @@
                set temp_value_drug.value_coded = drug.concept_id
              where temp_value_drug.value_drug = drug.drug_id;
 
+            -- Ignore for Liberia NCD encounters when value_coded is NULL
+            delete from temp_value_drug
+             where value_coded is NULL
```

Comment:
Don't we want to skip the NCD Intake and NCD FU for Liberia *entirely*.   On this form only value-coded are collected, right?  what if it possible that the value_drug does have that same primary key, and maps back to the another concept, although it was correct in the first place?

Reply:
You're right.  I thought of that, but didn't update the script.  Will fix.